w3c/aria-at-app
# Reduce permissions scopes needed by app

Reduces the GitHub permissions requested by the app, as seen below:

<img width="669" alt="GitHub authorization form showing the scope requested is only for the email address and list of organizations" src="https://user-images.githubusercontent.com/5255858/133472629-e3557e70-7a6b-4174-aebb-134d0985f719.png">

The invasive-sounding permissions previously requested by the app were due to GitHub requiring quite expansive permissions to support issue management. Therefore we decided to remove issue management from the app in favor of directly linking to the create issue page in GitHub. GitHub supports using query parameters to populate the issue page, so we can keep the prepopulated title and body, which have not changed.

## Testing Instructions

Note that I have deployed this version of the app to the Sandbox for testing!

- Go to the [Applications page](https://github.com/settings/applications?o=used-desc) in your GitHub settings.
- Find the version of the app for the environment you want to test, whether it is development (local) or the sandbox or other.
- Click the triple dot menu.
- Click revoke.
- Click accept.
- Now you can confirm the request permissions have been reduced!
